### ISSUE TITLE
Add „All orders revealed“ log entry

### ISSUE THREAD

**Issue by gereon77** [Longwelwind/swords-and-ravens#442]
It should show the state of the map with all units and their orders.

**Comment by Longwelwind** [comment]
Why "all units"? In my mind, we would only show each region with the order in it. The game log will be a bit big, though, but it's okay.

**Comment by gereon77** [comment]
I thought about a screenshot (really an image or dynamically created) which opens on hovering like the cards... If we see all units and orders it would give at least a bit the chance to replay the game...

**Comment by Longwelwind** [comment]
Ok, I see.

At first, I would only make a game log with each region and their orders. The units can be input inside the game log but not shown. Later we can support displaying a minimap with what you have in mind.

**Comment by gereon77** [comment]
This idea came to my mind as a user asked on discord what he did wrong. It’s hard to follow the events just by the text entries and without seeing the orders (passive orders like support or defense are never shown in game log). And I also think this feature would help debugging...
